src/components/Missions.js

```diff
@@ -21,45 +20,46 @@ function Missions() {
   const handleJoin = (id) => dispatch(joinMission(id));
   const handleLeave = (id) => dispatch(leaveMission(id));
 
-  console.log(missions)
+  console.log(missions);
   return (
     <div className="missions">
-      <table>
-        <thead key="1">
-          <tr>
-            <th className="tableHeader">Mission</th>
-            <th className="tableHeader">Description</th>
-            <th className="tableHeader">Status</th>
-            <th aria-label="join mission" />
-          </tr>
-        </thead>
-        <tbody>
+      <div>
+        <div key="1">
+          <ul>
+            <li className="tableHeader">Mission</li>
+            <li className="tableHeader">Description</li>
+            <li className="tableHeader">Status</li>
+            <li aria-label="join mission" />
+          </ul>
+        </div>
+        <div>
```

Comment:
- [ ] Let us remove the commented code in the code that has been pushed to keep your code clean.

![reactcap](https://user-images.githubusercontent.com/55635977/142480456-c1c52548-2155-49dc-868e-df509c4d9828.png)

- [ ] Don't disable the eslint errors, please fix the eslint errors without disabling them
- [ ] Kindly, use the arrow function (ES6 syntax) instead of the function.

![reactcap1](https://user-images.githubusercontent.com/55635977/142480477-17f12af4-4956-4526-a958-2c64fd7b4eb9.png)

- [ ] Please remove the `console.log`.
 - [ ] Kindly, use the arrow function (ES6 syntax) instead of the function.
 
![reactcap2](https://user-images.githubusercontent.com/55635977/142480483-7b5c1800-79b8-4541-9810-841a24efed08.png)

- [ ] Don't disable the eslint errors, please fix the eslint errors without disabling them
- [ ] Let us restructure your implementation here a little bit, instead of implementing both actions and reducer in the same file, I would recommend modifying the file by creating the action type, creator inside the action folder, and reducer inside the reducer folder.

![reactcap3](https://user-images.githubusercontent.com/55635977/142480497-208842d4-c0fe-44d7-8c76-20fa3636cc27.png)

- [ ] Don't disable the eslint errors, please fix the eslint errors without disabling them
- [ ] Let us restructure your implementation here a little bit, instead of implementing both actions and reducer in the same file, I would recommend modifying the file by creating the action type, creator inside the action folder, and reducer inside the reducer folder.

![reactcap4](https://user-images.githubusercontent.com/55635977/142480510-2ae62dcd-faf4-4693-b58e-f207d497d61a.png)